### ISSUE TITLE
clone时有没有超时机制?

### ISSUE THREAD

**Issue by d1y** [niudai/Github-Explorer#10]
大天朝的网络有时候崩坏, 根本就 `clone` 不下来, 导致一直等待, 很难受, 不知道有没有超时检测, 另外
在 `clone` 的时候推荐将 `github-url`  alias 为 淘宝的镜像站地址

原地址: https://github.com/d1y/github-to-cdn

镜像地址: https://github.com.cnpmjs.org/d1y/github-to-cdn

> 建议老哥写个设置, 让用户可选这个设置

以上, 感谢老哥

**Comment by niudai** [comment]
好的，这个建议不错

**Comment by niudai** [comment]
我问你个问题，你clone不下来吗？这个插件没有运行 git clone

**Comment by d1y** [comment]
> 我问你个问题，你clone不下来吗？这个插件没有运行 git clone

应该是运行了 `clone`, 这个插件是 `exec` - `git clone` 的方式吗, 应该是哪个库太大了, 我这边网络没科学上网

可不可以`hook`拿到进度条, 展示在 `bar` 上

**Comment by niudai** [comment]
不是 exec git clone，插件文档里一直在说 without git clone。。。 你把 git 卸载了也不影响插件运行

**Comment by d1y** [comment]
> 不是 exec git clone，插件文档里一直在说 without git clone。。。 你把 git 卸载了也不影响插件运行

那我就不知道了, 希望添加一个 `clone-bar` 显示速度之类的
